aten/src/ATen/native/cuda/Distributions.cu

```diff
@@ -82,7 +82,7 @@ void gamma_cuda_kernel(
         };
         BaseSampler<accscalar_t, decltype(normal_lambda)> standard_normal(normal_lambda);
         auto sample = sample_gamma<scalar_t, accscalar_t, decltype(uniform_lambda), decltype(normal_lambda)>(alpha, standard_uniform, standard_normal);
-        auto min_value = std::numeric_limits<scalar_t>::lowest();
+        auto min_value = std::numeric_limits<scalar_t>::min();
```

Comment:
I think this was a mistake - since `lowest` would be a [negative value](https://en.cppreference.com/w/cpp/types/numeric_limits/lowest). I was seeing 0s on CUDA.